### DISCUSSION TITLE
Can UC bypass CF on aws ec2 or digitalocean droplet?

### DISCUSSION THREAD

**Discussion by mutiinsani** [ultrafunkamsterdam/undetected-chromedriver#1393] [Q&A]
Hi, I'm wondering has anyone tried UC on those servers?
If yes, do we need specific settings for it? I tried to use UC out of the box but it cannot seem to pass cloudflare.
Code for testing.

```python
import undetected_chromedriver as uc
import time

options = uc.ChromeOptions()
options.add_argument('--headless')
driver = uc.Chrome(options=options)

driver.get("https://nowsecure.nl")
time.sleep(5)
print(driver.title)
driver.quit()
``` 
Expected output is "nowSecure" but I keep getting "Just a moment..."

I've tried with Python version 3.9 and 3.10, latest Chrome and latest UC.

Please any response is much appreciated, thank you!

**Comment by maroun2** [comment]
I have same issue, about a 3 days back UC stop working for me for nowsecure.nl (cloudforge). I tried multiple IPs and update to latest UC and still do not work.

**Reply by maroun2** [reply]
related to https://github.com/ultrafunkamsterdam/undetected-chromedriver/discussions/1387

**Reply by mutiinsani** [reply]
does it work for you @maroun2 ?

**Reply by maroun2** [reply]
open dev tab works for me

**Reply by damodharheadrun** [reply]
open dev tab works me as well

**Comment by ultrafunkamsterdam** [comment]
Working from a datacenter ip ( or vpn for that matter ), will always get you the the extended test. just verified some minutes ago

for cloudflare to pass currently, you need to be on a "somewhat" home ip address (or a small company unknown), 
and use 
```
with driver:
    driver.get('https://nowsecure.nl')
```
